src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -25,6 +30,8 @@
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Web" />
     <Reference Include="System.ServiceModel" />
+
+    <ProjectReference Include="..\Datadog.Trace.AspNet\Datadog.Trace.AspNet.csproj" />
```

Comment:
super nit: This could be moved up to the other ProjectReference node for better readability

Reply:
I put it here so it's only included as a dependency in `net45` and `net461`. I suppose it wouldn't really hurt to have it always, if you prefer. `Datadog.Trace.AspNet` just won't have any content targeting `netstandard2.0`.

Reply:
Oh, silly me. I forgot to look up and see that this ItemGroup had the `TargetFramework` check. Disregard my comment! It looks good 👍